recipes/valgrind/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e -o pipefail
+
+./configure --prefix=${PREFIX} --disable-dependency-tracking --enable-only64bit
+
+SYSLWR=`uname | tr '[:upper:]' '[:lower:]'`
+
+mv ${SRC_DIR}/coregrind/link_tool_exe_${SYSLWR} ${SRC_DIR}/coregrind/link_tool_exe_${SYSLWR}_orig
+echo '#!/usr/bin/env perl' >> ${SRC_DIR}/coregrind/link_tool_exe_${SYSLWR}
+cat ${SRC_DIR}/coregrind/link_tool_exe_${SYSLWR}_orig >> ${SRC_DIR}/coregrind/link_tool_exe_${SYSLWR}
+chmod u+x ${SRC_DIR}/coregrind/link_tool_exe_${SYSLWR}
+
+make
```

Comment:
Sort of. There is `make regtest`. However it sounds like this will be a pain (see below).

Would rather add specific tests to the recipe. There is already a test of `valgrind` on a simple program (i.e. `ls`).

> To test the results, issue: make regtest. The tests may hang forever if GDB-8.1 is not installed. Some tests are known to hang also, depending on the version of glibc. Problematic tests can be disabled by changing the prereq: line in the corresponding .vgtest file to prereq: false.

ref: http://www.linuxfromscratch.org/blfs/view/svn/general/valgrind.html